src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -536,4 +536,12 @@ public uint GetPaginationLimit(int? first)
             return defaultPageSize;
         }
     }
+
+    /// <summary>
+    /// Retrieve the value of LogLevel from runtime
+    /// </summary>
+    public Level? LogLevelValue()
```

Comment:
```suggestion
    public Level? LogLevel()
```
nit: "value" is implied.

Reply:
How was this resolved? if you decide not to make any changes, please add follow up comment indicating which direction you chose to follow and your deciding factors.

Reply:
Changed to IsLogLevel